catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/XstreamPathConverter.java

```diff
@@ -111,7 +112,8 @@ protected void updatePath(
   /**
    * This method uses a streaming-like approach to compare 2 paths with a single iteration. The
    * comparison excludes count indexes in the path as well as the value of attributes in determining
-   * equivalence. For the purposes of this method, "/a/b/c" matches "/a/b[2]/c/@attr"
+   * equivalence. For the purposes of this method, "/a/b/c" matches "/a/b[2]/c/@attr" Also checks
```

Comment:
Should there be a period before " Also"?

Reply:
Fixed